src/test/java/com/google/sps/data/ScheduledInterviewServletTest.java

```diff
@@ -54,143 +54,218 @@
   LocalServiceTestHelper helper =
       new LocalServiceTestHelper(new LocalCapabilitiesServiceTestConfig());
   private FakeScheduledInterviewDao scheduledInterviewDao;
+  private FakeAvailabilityDao availabilityDao;
   private FakePersonDao personDao;
 
-  @Before
-  public void setUp() {
-    helper.setUp();
-    scheduledInterviewDao = new FakeScheduledInterviewDao();
-    personDao = new FakePersonDao();
-  }
-
-  @After
-  public void tearDown() {
-    helper.tearDown();
-  }
-
-  private final Person interviewer =
+  private final Person person1 =
       Person.create(
-          emailToId("user@company.org"),
-          "user@company.org",
-          "User1",
-          "Subject",
+          emailToId("user1@mail"),
+          "user1@mail",
+          "User",
+          "Test",
           "Google",
           "SWE",
           "linkedIn",
           EnumSet.noneOf(Job.class));
+  private final Availability person1Avail1 =
+      Availability.create(
+          person1.id(),
+          new TimeRange(
+              Instant.parse("2020-07-20T12:45:00Z"), Instant.parse("2020-07-20T13:00:00Z")),
+          -1,
+          false);
+  private final Availability person1Avail2 =
+      Availability.create(
+          person1.id(),
+          new TimeRange(
+              Instant.parse("2020-07-20T13:00:00Z"), Instant.parse("2020-07-20T13:15:00Z")),
+          -1,
+          false);
+  private final Availability person1Avail3 =
+      Availability.create(
+          person1.id(),
+          new TimeRange(
+              Instant.parse("2020-07-20T13:15:00Z"), Instant.parse("2020-07-20T13:30:00Z")),
+          -1,
+          false);
+  private final Availability person1Avail4 =
+      Availability.create(
+          person1.id(),
+          new TimeRange(
+              Instant.parse("2020-07-20T13:30:00Z"), Instant.parse("2020-07-20T13:45:00Z")),
+          -1,
+          false);
 
-  private final Person interviewee =
+  private final Person person2 =
       Person.create(
-          emailToId("user@gmail.com"),
-          "user@gmail.com",
-          "User2",
-          "Subject",
+          emailToId("user2@mail"),
+          "user2@mail",
+          "User",
+          "Test",
           "Google",
           "SWE",
           "linkedIn",
           EnumSet.noneOf(Job.class));
+  private final Availability person2Avail1 =
```

Comment:
These names are pretty bad.

Is there an interesting property that these instances can each be named after?

Reply:
Honestly the only interesting property is the person. All 3 availability set are the same apart from the userId. I started renaming and realized that it didn't make much sense.

Reply:
If these don't have a distinct identifying property then they likely don't benefit at all from being factored out here. I'd argue that they're actually harming readability, since it's not obvious at all in the usage sites what specifically these values represent.

The tradeoffs in tests are slightly different than regular code. In tests, the priority is that the code is self-evident, and often that means that you should inline and permit additional duplication where it enhances clarity.

Reply:
I renamed the Person instances which I think helps bring some clarity.